# I have checked all headers and don't think the shared library
# bump is a needed one.  We patch it back to prevent large
# rebuild need.
--- build/version.var.orig	2011-03-25 14:08:12.000000000 -0700
+++ build/version.var	2011-03-28 01:23:08.215863119 -0700
@@ -17,7 +17,7 @@
 ol_minor=4
 ol_patch=25
 ol_api_inc=20425
-ol_api_current=9
+ol_api_current=8
 ol_api_revision=0
 ol_api_age=7
 ol_release_date="2011/03/26"
